src/index.js

```diff
@@ -0,0 +1,199 @@
+"use strict";
+
+let temperature = 73;
+
+const tempDisplay = document.getElementById("temperature");
+const tempUpBtn = document.getElementById("temp-up");
+const tempDownBtn = document.getElementById("temp-down");
+const groundEmoji = document.getElementById("ground-emoji");
+
+const cityInput = document.getElementById("city-input");
+const currentCityDisplay = document.getElementById("current-city");
+const realTimeBtn = document.getElementById("get-realtime-btn");
+const resetBtn = document.getElementById("reset");
+
+const DEFAULT_CITY = "New York City";
+const PROXY_SERVER_BASE_URL = 'http://127.0.0.1:5000';
+
+
+// Wave2
+const updateTemperatureVisuals = () => {
+  const tempC = Math.round((temperature - 32) * 5 / 9);
+  tempDisplay.innerHTML = `${temperature}&deg;F<br>⎯<br>${tempC}&deg;C`;
+
+  if (temperature >= 80) {
+    tempDisplay.style.color = "hotpink";
+    groundEmoji.textContent = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+  } else if (temperature >= 70) {
+    tempDisplay.style.color = "pink";
+    groundEmoji.textContent = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+  } else if (temperature >= 60) {
+    tempDisplay.style.color = "yellow";
+    groundEmoji.textContent = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+  } else if (temperature >= 50) {
+    tempDisplay.style.color = "cyan";
+    groundEmoji.textContent = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+  } else {
+    tempDisplay.style.color = "teal";
+    groundEmoji.textContent = "⛄️⛄️⛄️";
+  }
+};
+
+
+// changing temperature buttons
+tempUpBtn.addEventListener("click", () => {
+  temperature += 1;
+  updateTemperatureVisuals();
+});
+
+tempDownBtn.addEventListener("click", () => {
+  temperature -= 1;
+  updateTemperatureVisuals();
+});
+
+updateTemperatureVisuals();
+
+// Wave 3
+const updateLocationCard = () => {
+  if (currentCityDisplay && cityInput) {
+    let cityText = cityInput.value.trim(); 
+
+    if (cityText.length > 0 && cityText.charAt(cityText.length - 1) !== ',') {
```

Comment:
Instead of checking for if the length of the check is greater than zero, you could just check if the truthiness.
```suggestion
    if (cityText.length && cityText.charAt(cityText.length - 1) !== ',') {
```